server/src/tests/integration.rs

```diff
@@ -435,13 +436,47 @@ fn run_op() -> Result<(Child, Child, Child)> {
 async fn use_optimism_bridge() -> Result<()> {
     pause(Some(Duration::from_secs(OP_START_IN_SECS)));
 
-    deposit_to_l2().await?;
+    deposit_eth_to_l2().await?;
+
+    deposit_erc20_to_l2().await?;
+
     withdrawal::withdraw_to_l1().await?;
 
     Ok(())
 }
 
-async fn deposit_to_l2() -> Result<()> {
+async fn deposit_erc20_to_l2() -> Result<()> {
```

Comment:
nit: We can move this method below `deposit_eth_to_l2` to match the order of calls.

Reply:
Done in https://github.com/MovedNetwork/op-move/pull/307/commits/3de586aa4b6401bee698efda703acb6428e7379d